platform/docs/docs/platform/services/data/SyncGroupService.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 8
+sidebar_label: SyncGroup Service
+---
+
+# Sync Group Service
+
+## Overview
+
+The `SyncGroupService` is responsible for managing synchronization groups in the OHIF Viewer. Synchronization groups allow multiple viewports to be synchronized based on various criteria, such as camera position, window level, zoom/pan, and image slice position. This service provides a centralized way to create, update, and manage synchronization groups.
+
+Right now, synchronization groups can be defined in the hanging protocols or manually assigning buttons.
+
+
+
+
+## API
+
+- `getSyncCreatorForType(type)`: Returns the synchronizer creator function for the specified type.
+- `addSynchronizerType(type, creator)`: Adds a new synchronizer type with a custom creator function.
+- `getSynchronizer(id)`: Retrieves a synchronizer by its ID.
+- `getSynchronizersOfType(type)`: Retrieves an array of synchronizers of the specified type.
+- `addViewportToSyncGroup(viewportId, renderingEngineId, syncGroups)`: Adds a viewport to one or more synchronization groups.
+- `destroy()`: Destroys all synchronizers.
+- `getSynchronizersForViewport(viewportId)`: Retrieves an array of synchronizers associated with the specified viewport.
+- `removeViewportFromSyncGroup(viewportId, renderingEngineId, syncGroupId?)`: Removes a viewport from a specific synchronization group or all synchronization groups if no group ID is provided.
+
+## Usage
+### Via hanging protocols
+You can set up different types of synchronization groups for your viewports. For example, in the TMTV hanging protocol (`extensions/tmtv/src/getHangingProtocolModule.js`), we can see how different synchronization groups are defined for various viewports:
+
+```javascript
+const ptAXIAL = {
+  viewportOptions: {
+    // ...
+    syncGroups: [
+      {
+        type: 'cameraPosition',
+        id: 'axialSync',
+        source: true,
+        target: true,
+      },
+      {
+        type: 'voi',
+        id: 'ptWLSync',
+        source: true,
+        target: true,
+      },
+      {
+        type: 'voi',
+        id: 'ptFusionWLSync',
+        source: true,
+        target: false,
+        options: {
+          syncInvertState: false,
+        },
+      },
+    ],
+  },
+  // ...
+};
+```
+
+
+
+In this example, the `ptAXIAL` viewport is part of three synchronization groups:
+
+1. `cameraPosition` group with the ID `'axialSync'`: This group synchronizes the camera position across viewports that are both source and target.
+2. `voi` (Window Level) group with the ID `'ptWLSync'`: This group synchronizes the window level settings across viewports that are both source and target.
+3. `voi` group with the ID `'ptFusionWLSync'`: This group synchronizes the window level settings, but the `ptAXIAL` viewport is only a source, not a target.
+
+
+### Manually through a button
```

Comment:
Can you explain here that you can also create them through the hanging protocol, and then control the state via a toolbar button?  That is a fairly important use case as it allows defining exactly which set of viewports have the sync group turn on for.